packages/scoped-elements/src/scoped-elements.js

```diff
@@ -0,0 +1,28 @@
+import { createUniqueTag } from "./tag.js";
+
+const registeredElements = new Map();
```

Comment:
This library will very likely be loaded multiple times per runtime, which means there is no single Map and we will end up with multiple elements defined per class. Do we think that's a problem? Just some food for thought. We could create a `Map` on the window if we really want to share it.

Reply:
Even if it's loaded multiple times it's going to be evaluated just once (http://www.ecma-international.org/ecma-262/6.0/#sec-moduleevaluation). However, I didn't think that different versions could coexist at the same time by different application parts, so at the end it would be evaluated several times (once per version used).

Good point! Thanks @LarsDenBakker

Reply:
People will use this in a micro front end setup, where separate parts are bundled and dependencies duplicated. So even the same version may end up on the page multiple times.